pkg/oc/lib/newapp/cmd/newapp_test.go

```diff
@@ -197,24 +202,28 @@ func TestBuildTemplates(t *testing.T) {
 		appCfg.ClassificationWinners = map[string]ArgumentClassificationWinner{}
 
 		// the previous fake was broken and didn't 404 properly.  this test is relying on that
-		templateFake := templatefakeclient.NewSimpleClientset()
-		imageFake := imagefakeclient.NewSimpleClientset()
+		imageFake := imagev1fake.FakeImageV1{Fake: &(imagefakeclient.NewSimpleClientset()).Fake}
+		templateFake := templatev1fake.FakeTemplateV1{Fake: &(templatefakeclient.NewSimpleClientset()).Fake}
+		routeFake := routev1fake.FakeRouteV1{Fake: &(routefakeclient.NewSimpleClientset()).Fake}
+
+		customScheme := runtime.NewScheme()
```

Comment:
Using `apitesting.SchemeForOrDie` like here https://github.com/openshift/origin/blob/ebc5e89575ca09ccb48d3dfc08c67eb95ef795be/pkg/image/controller/scheduled_image_controller_test.go#L57
is preferred.